solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc

```diff
@@ -277,7 +277,7 @@ For example:
 
 [source,text]
 ----
-q=supervillians&debugQuery=on&explainOther=id:juggernaut
+q=supervillains&debugQuery=on&explainOther=id:juggernaut
```

Comment:
Just a quick, this query works now?    Wanted to make sure the typo doesn't actually need to be there!

Reply:
Never mind, I check, and now I get it!